client/src/components/changePasswordForm.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import toastr from 'toastr';
+import NavigationBar from './NavigationBar';
+import changePassword from '../actions/changePassword';
+import '../styles/passwordchange.scss';
+/**
+ *
+ *
+ * @class ChangePasswordForm
+ *
+ * @extends {Component}
+ */
+
+class ChangePasswordForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      oldPassword: '',
+      password: ''
+    }
+  }
+
+  onNameChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  onSubmit = (event) => {
+    event.preventDefault();
+    this.props.changePassword(this.state).then((res) => {
+      toastr.success('Password successfully changed')
+      this.setState({
+        oldPassword: '',
+        password: ''
+      })
+    })
+    .catch((error) => {
+      toastr.error(error.response.data.error);
+    })
+  }
+  render() {
+    return (
+      <div >
+        <NavigationBar />
+        <div>
+        <form className="mx-auto app-login-form card-login">
```

Comment:
### Nitpick

This should be indented inwards.
Pay attention to indenting wrapped html tags.